operations/helm/charts/mimir-distributed/CHANGELOG.md

```diff
@@ -29,6 +29,9 @@ Entries should include a reference to the Pull Request that introduced the chang
 
 ## main / unreleased
 
+* [ENHANCEMENT] Dashboards: allow switching between using classic of native histograms in dashboards. #7627 #7674
```

Comment:
`allow switching between use of classic or native histograms in dashboards.`
or 
`allow switching between using classic or native histograms in dashboards.`